### DISCUSSION TITLE
Display if installed HA OS is 32 or 64 bit

### DISCUSSION THREAD

**Discussion by swerveshot** [home-assistant/frontend#11410] [Configuration] [answered]
I am migrating the tutorial my HASSIO install from SD Card to USB using the [Everything Smart Home tutorial](https://www.youtube.com/watch?v=QxtDyMbDOh4). This migration involves re-installing a fresh copy of HASSIO to an USB drive. 

Unfortunately I could not remember if I used the 32 or 64 bit image on the original install. Although this can easily be queried using the CLI this might be a challenge for users with little linux experience. So why not include it in the hassio/system screen where the OS version is displayed?

![image](https://user-images.githubusercontent.com/10358044/150674250-328a8103-91c9-4a19-b9c4-88b0af6f0512.png)

**Comment by frenck** [comment, marked as answer]
[![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/)

![image](https://user-images.githubusercontent.com/195327/150674403-8666b527-d502-4651-baeb-339fe720dfa1.png)

**Reply by swerveshot** [reply]
Awesome! Thanks for the feedback mate. 👍🏻
